lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -323,8 +332,9 @@ def self.available_options
                               UI.user_error!("You can't use 'ipa' and '#{value.key}' options in one run")
                             end,
                               verify_block: proc do |value|
-                                accepted_formats = [".ipa"]
-                                UI.user_error!("Only \".ipa\" formats are allowed, you provided \"#{File.extname(value)}\"") unless accepted_formats.include? File.extname(value)
+                                accepted_formats = [".ipa", ".app", ".app.zip"]
```

Comment:
Like @evkhramkov said we may want to use `app` instead of reusing `ipa`.
That being said I think we should have a generic field for build archives, as we plan on introducing `dmg` and `pkg` as well in the near future. See #112